util/execution/req.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package execution
+
+import "github.com/pingcap/tidb/util/chunk"
+
+// ExecRequest is input parameter of Executor.Next` method.
+type ExecRequest struct {
```

Comment:
Should it put into `util` rather than `executor`?

Reply:
emm.. at first I had to try put it into executor, but I found this `req` not only used in executor but also some other package, so it doesn't belong to `executor` in strict, so at last I'm following `Chunk` does to put it here.

Reply:
can we put this struct into "util/execdetails"?

Reply:
`execdetail` just like a collection info package...In my understand.... :rofl:

Reply:
```
        imports github.com/pingcap/tidb/ddl
	imports github.com/pingcap/tidb/ddl/util
	imports github.com/pingcap/tidb/kv
	imports github.com/pingcap/tidb/util/execdetails
	imports github.com/pingcap/tidb/util/chunk
	imports github.com/pingcap/tidb/types
	imports github.com/pingcap/tidb/sessionctx/stmtctx
	imports github.com/pingcap/tidb/util/execdetails
```

Reply:
@zz-jason add to execdetails will got import cycle

Reply:
or let's put it into chunk package? @zz-jason  @winoros

Reply:
:joy: Oh `stmtctx` imports `execdeail`

Put it into `chunk` and name it `ChunkRequest`?